cmd/csi-resizer/main.go

```diff
@@ -42,6 +44,7 @@ var (
 	showVersion = flag.Bool("version", false, "Show version")
 
 	enableLeaderElection      = flag.Bool("leader-election", false, "Enable leader election.")
+	leaderElectionType        = flag.String("leader-election-type", "endpoints", "the type of leader election, options are 'endpoints' (default) or 'lease' (strongly recommended)")
```

Comment:
resizer is still alpha (and not officially released yet). Let's just start from lease :-)

Reply:
gladly :)

Reply:
Going to identity, retry, lease duration & renew flags. Easier to add later than to remove